pkgs/servers/dns/pdns/default.nix

```diff
@@ -53,10 +54,12 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   # Configure phase requires 64-bit time_t even on 32-bit platforms.
-  env.NIX_CFLAGS_COMPILE = toString (lib.optionals stdenv.hostPlatform.is32bit [
-    "-D_TIME_BITS=64"
-    "-D_FILE_OFFSET_BITS=64"
-  ]);
+  env.NIX_CFLAGS_COMPILE = toString (
+    lib.optionals stdenv.hostPlatform.is32bit [
+      "-D_TIME_BITS=64"
+      "-D_FILE_OFFSET_BITS=64"
+    ]
+  );
```

Comment:
I wonder if we can do something about this. https://github.com/NixOS/nixpkgs/issues/79303

Reply:
Replaced this with an upstream configure option.